.eslintrc

```diff
@@ -101,7 +101,8 @@
 			},
 			"rules": {
 				"prettier/prettier": "warn",
-				"max-nested-callbacks": "off" // unit tests involve a lot of nested callbacks
+				"max-nested-callbacks": "off", // unit tests involve a lot of nested callbacks
+				"@typescript-eslint/unbound-method": "off" // unit tests also involve passing unbounded methods
```

Comment:
I disagree. Messing with method binding can cause JS-land `this`-binding weirdness that I'd rather ESLint catch by default. Typically, if ESLint throws this at you, you should redesign to either call the method in a lambda, or build a better method. If you _must_ hand around an unbound method, you shouldn't.

But if for sure you really promise it won't break anything ever, then throw an `eslint-ignore-next-line` where needed to silence the warning at the use site.